build.gradle

```diff
@@ -0,0 +1,36 @@
+/*
+ Applies core Gradle plugins, which are ones built into Gradle itself.
+*/
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+}
+
+plugins {
+    // Java for compile and unit test of Java source files. Read more at:
+    // https://docs.gradle.org/current/userguide/java_plugin.html
+    id 'java'
+
+    // Aggregate Javadoc generation. Read more at:
+    // https://docs.freefair.io/gradle-plugins/6.6.1/reference/#_io_freefair_javadocs
+    id 'io.freefair.javadocs' version "6.6.3"
```

Comment:
Oh, I copied and pasted the link include from one of the original place we included the plugin and didn't notice the doc link. We might want to make that a higher level link so the version isn't included.

This sort of looks like a downgrade but it's not too. The `8.0.1` has a beta-ish version of aggregate javadocs that makes breaking changes. The gradle project, specifically with Jacoco can't build with `8.0.1` but works fine with `6.6.3`. I'm logging a ticket for the larger issue of `8.0.1` because it says aggregate-javadocs shouldn't be used on java projects. It turned into a way bigger problem than the scope of this ticket. I haven't been able to find any documentation on how the new version is supposed to work and I spent over a day trying to solve the problem.

If you want to do a quick call for me to show you, I can set something up.

Reply:
Ok, but you can't just downgrade versions and expect them to stay that way. Dependabot will automatically upgrade everything. If there's a particular dependency that can't be updated, it needs to be added to Dependabot's ignore list and a mechanism to go back and upgrade it in a reasonable time frame created.

Reply:
I figured dependabot wouldn't be able to merge since the build would fail. I will figure out how to make dependabot ignore this for now and add that to this PR.

Reply:
I found it! `io.freefair.javadoc* 8.0.*` needs gradle version 8. It looks like we're using Gradle 8 for the main build but when Jacoco runs it's not using version 8 so `io.freefair.javadoc*` with version > `6.6.3` doesn't work. I'm adding it to the ignore list for now and will look into how Jacoco is running next week. @pnwamk FYI.